src/app/src/components/OARMap.jsx

```diff
@@ -113,26 +116,28 @@ class OARMap extends Component {
             fetching,
             error,
             data,
+            singleFacilityData,
         } = this.props;
 
         if (fetching) {
-            return null;
+            return noop();
```

Comment:
Why are we returning the result of calling noop here? Why not return `undefined` or keep the `null` return?

Reply:
I think I used this once and then decided to replace all of `null` with `noop`. However, for consistency's sake it'd be better to standardize on returning null here.

Reply:
Changed back to null in 868953dfba001c3275bfe3b1193ad80a98adee00